sys/include/uuid.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_uuid
+ * @ingroup     sys
+ * @brief       Generate system wide unique IDs
```

Comment:
but now that I thought about it, let's do it like that: I add that comment you requested and if we find a better name, we can refactor then.